bbq/vm/vm.go

```diff
@@ -810,6 +810,16 @@ func opSetUpvalue(vm *VM, ins opcode.InstructionSetUpvalue) {
 	}
 }
 
+func opCloseUpvalue(vm *VM, ins opcode.InstructionCloseUpvalue) {
+	absoluteLocalsIndex := int(vm.callFrame.localsOffset) + int(ins.Local)
+	openUpvalues := vm.callFrame.openUpvalues
+	upvalue := openUpvalues[absoluteLocalsIndex]
+	if upvalue != nil {
+		upvalue.closed = vm.locals[absoluteLocalsIndex]
+		delete(openUpvalues, absoluteLocalsIndex)
```

Comment:
Not sure, could deleting the value (as opposed to just clearing / setting `nil`) affect the indexes for the rest of the up-values?

Reply:
I see. Thanks for the explanation!